toolbox/include/CentroidalMomentum.h

```diff
@@ -1,35 +1,36 @@
 #ifndef WBT_CENTROIDALMOMENTUM_H
 #define WBT_CENTROIDALMOMENTUM_H
 
-#include "WBIModelBlock.h"
+#include "WBBlock.h"
 
 namespace wbt {
     class CentroidalMomentum;
 }
 
-class wbt::CentroidalMomentum : public wbt::WBIModelBlock {
+namespace iDynTree {
+    class SpatialMomentum;
+}
 
-    double *m_basePose;
-    double *m_centroidalMomentum;
+class wbt::CentroidalMomentum : public wbt::WBBlock
+{
+private:
+    iDynTree::SpatialMomentum* m_centroidalMomentum;
 
-    //input buffers
-    double *m_basePoseRaw;
-    double *m_configuration;
-    double *m_baseVelocity;
-    double *m_jointsVelocity;
+    static const unsigned INPUT_IDX_BASE_POSE;
```

Comment:
Just a couple of comments on this:
- If you want to adopt the CamelCase style instead of C-Style Macro style for me is ok
- what are the pro/cons of private static const variables vs static global variables in the Cpp file?

Reply:
> If you want to adopt the CamelCase style instead of C-Style Macro style for me is ok

I add this as Post-WB3.0 task, I have nothing against it.

> What are the pro/cons of private static const variables vs static global variables in the Cpp file?

No big differences, actually moving them in the cpp keeps headers cleaner.